### PR TITLE
Add Player::Get(Side) overload

### PR DESCRIPTION
**Pull request by iRebbok**

### CODE REVIEW

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -840,6 +840,13 @@ public int Scp330Usages
         /// </summary>
         internal static ConditionalWeakTable<ReferenceHub, Player> UnverifiedPlayers { get; } = new ConditionalWeakTable<ReferenceHub, Player>();
 
+        /// <summary>
+        /// Gets a <see cref="Player"/> <see cref="IEnumerable{T}"/> filtered by side.
+        /// </summary>
+        /// <param name="side">The player's side.</param>
```
Comment (BuildBoy12) [inline Exiled.API/Features/Player.cs:846]: Change player's to players' for consistency with other IEnumerable Gets.

**Review by BuildBoy12 — APPROVED** [review]

File: Exiled.API/Features/Player.cs (hunk 1 above)
Comment (iRebbok) [reply Exiled.API/Features/Player.cs:846]: Sure, I didn't notice it.

**Review by BuildBoy12 — APPROVED** [review]

File: Exiled.API/Features/Player.cs [hunk 2]
```diff
@@ -840,6 +840,13 @@ public int Scp330Usages
         /// </summary>
         internal static ConditionalWeakTable<ReferenceHub, Player> UnverifiedPlayers { get; } = new ConditionalWeakTable<ReferenceHub, Player>();
 
+        /// <summary>
+        /// Gets a <see cref="Player"/> <see cref="IEnumerable{T}"/> filtered by side.
+        /// </summary>
+        /// <param name="side">The players' side.</param>
+        /// <returns>Returns the filtered <see cref="IEnumerable{T}"/>.</returns>
+        public static IEnumerable<Player> Get(Side side) => List.Where(player => player.Side == side);
```
Comment (galaxy119) [inline Exiled.API/Features/Player.cs:848]: .Where() :vomiting_face:

**Review by galaxy119 — APPROVED** [review]

File: Exiled.API/Features/Player.cs (hunk 2 above)
Comment (BuildBoy12) [reply Exiled.API/Features/Player.cs:848]: Isn't the performance impact of .Where versus the foreach variable, as shown by https://dotnetfiddle.net/pG4kKi?

Comment (iopietro) [reply Exiled.API/Features/Player.cs:848]: We're talking about differences of microseconds and the player list will always have less than 100 objects.
Also, that query will only be evaluated when iterated by a foreach, Count(), ToList() or manually.

**Review by iopietro — APPROVED** [review]

File: Exiled.API/Features/Player.cs (hunk 2 above)
Comment (iRebbok) [reply Exiled.API/Features/Player.cs:848]: `Where(Predicate<T>)` doesn't go through all values and doesn't allocate a new array, it just returns a new instance of 'filter' with a predicate passed by a developer. So there's no reason to replace it with a custom 'filter' implementation.

**Pull request merged**